assets/armsnippets.jsonc

```diff
@@ -985,9 +985,7 @@
             "               \"resource\":{",
             "                   \"id\": \"${3:table-name}\"",
             "               },",
-            "            \"options\":{",
-            "                \"throughput\": ${4:400}",
-            "            }",
+            "            \"options\":{}",

```

Comment:
Not so this is so expensive but still probably an important option to keep front and center

Reply:
Fixed